internal/sourcemap/metadata.go

```diff
@@ -196,7 +186,7 @@ func (s *MetadataCachingFetcher) sync(ctx context.Context) error {
 		return err
 	}
 
-	var ms []Metadata
+	ms := make(map[Identifier]Metadata, len(body.Hits.Hits))
```

Comment:
What about making it `map[Identifier]string`?

Reply:
Good point! Updated :+1: